docs/deployment/settings.md

```diff
@@ -122,9 +122,18 @@ You can configure the Kyuubi properties in `$KYUUBI_HOME/conf/kyuubi-defaults.co
 ## Kyuubi Configurations
 
 #
-# kyuubi.authentication           NONE
```

Comment:
We may give a much more practical example.

The current example is from the template directly, where all examples for individual configs are commented out, while we shall give meaningful ones on the contrary.

Or, for legible, we could delete this entire block here. For example, `You can configure the Kyuubi properties in $KYUUBI_HOME/conf/kyuubi-defaults.conf, see $KYUUBI_HOME/conf/kyuubi-defaults.conf.template as an example,. `